locale/Makefile

```diff
@@ -54,6 +54,7 @@ tx-update: tx-pull
 
 mo-files: $(MOFILES)
 	git add $(POFILES) $(POTFILE) ../locale/*/LC_MESSAGES
+	[[ ! -f locale/action_names.rb ]] || git add locale/action_names.rb
```

Comment:
@adamruzicka do we even need to track this file in git? If we ensure it's generated as part of the job then we can add it to `.gitignore` too, right?

Reply:
The only thing that I can think of is that the `*.pot` files mention from which file the string to be translated comes from. If we treat the action names file as ephemeral, then the pots will point to a missing file. Apart from that, I don't think it has any value

Reply:
Then I'm leaning to ignoring it. Otherwise I'd need to spend time to make it reproducible (so at least sort it). If we want to help translators, we can add comments. From https://projects.theforeman.org/projects/foreman/wiki/Translating#Helping-translators

> If you want to leave a note to the translator, just drop a comment before the string in the format of # TRANSLATORS: your comment here

We could add something like `# TRANSLATORS: comes from action $action`.

I'll update this PR to ignore it instead and do a bulk update on plugins where it is currently in git.

Reply:
Updated